.ci/clang.sh

```diff
@@ -15,13 +20,14 @@ if [ "$TRAVIS_OS_NAME" = "linux" ]; then
 	export CC=/usr/local/clang-$clang_version/bin/clang
 	symbolizer=/usr/local/clang-$clang_version/bin/llvm-symbolizer
 	export ASAN_SYMBOLIZER_PATH=$symbolizer
+	export MSAN_SYMBOLIZER_PATH=$symbolizer
```

Comment:
I think the variable has to be named `ASAN_SYMBOLIZER_PATH` (same for MSAN) for Clang to pick it up.

http://clang.llvm.org/docs/MemorySanitizer.html

> Make sure that llvm-symbolizer binary is in PATH, or set environment variable MSAN_SYMBOLIZER_PATH to point to it.

Or we could maybe change the PATH, but looks like `$symbolizer` is used in some other variables as well, so we can't get rid of it. So rename `$symbolizer` to `SAN_SYMBOLIZER_PATH`?